skins/laika/src/scss/overrides.scss

```diff
@@ -29,6 +42,19 @@
 	.check {
 		font-size: 12px;
 	}
+	.control-label {
+		padding-left: 18px;
+	}
+}
+
+.b-checkbox.checkbox {
+	.control-label {
+		padding-left: 18px;
+	}
+	input[type="checkbox"] + .check {
+		border: 1px solid $fun-color-dark;
+		border-radius: 2px;
```

Comment:
Instead of this rule, you can set the `$radius` variable to 2px which should be generally fine for our purposes. See: `skins/laika/node_modules/buefy/src/scss/components/_checkbox.scss`.